app/workers/clean_up_course_options_with_no_study_mode.rb

```diff
@@ -0,0 +1,23 @@
+class CleanUpCourseOptionsWithNoStudyMode
+  include Sidekiq::Worker
+
+  def perform
+    CourseOption.where('study_mode = \'0\'').each do |nil_co|
```

Comment:
One I wanted to double check. Could there be other invalid values in the database? e.g. `study_move == '1'` etc.?

As a sanity check I would be tempted to check for any values that are not `IN ('full_time', 'part_time')`

If there are part_time courses in the system then I guess we need to handle those too, otherwise what you have is fine.

Reply:
I would change `each` to `find_each` when processing a big dataset like this to avoid loading all the models into memory at once but it's probably not big enough to worry about here.

Reply:
Historically there shouldn't be any part time courses, but you're right. I've updated the code to handle 'nil' part time courses, too ('1').